packages/guile/guile.1.0/opam

```diff
@@ -0,0 +1,45 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings to GNU Guile Scheme for OCaml"
+description:
+  "The guile library is Free Software high-level OCaml bindings to GNU Guile 3.0, supporting easy interop between OCaml and GNU Guile Scheme."
+maintainer: ["kirang@comp.nus.edu.sg"]
+authors: ["Kiran Gopinathan"]
+license: "GPL-3.0+"
+homepage: "https://github.com/gopiandcode/guile-ocaml"
+bug-reports: "https://github.com/gopiandcode/guile-ocaml/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "conf-guile" {>= "1"}
```

Comment:
Just to prevent the following error in Alpine (probably a bug in the alpine package):
```
#=== ERROR while compiling guile.1.0 ==========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.14.0 | pinned(https://github.com/gopiandcode/guile-ocaml/archive/1.0.tar.gz)
# path                 ~/.opam/4.14/.opam-switch/build/guile.1.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p guile -j 31 --promote-install-files=false @install
# exit-code            1
# env-file             ~/.opam/log/guile-7-e83d37.env
# output-file          ~/.opam/log/guile-7-e83d37.out
### output ###
# File "stubgen/dune", line 12, characters 0-263:
# 12 | (rule (targets bindings_stubs_gen.exe)
# 13 |  (deps bindings_stubs_gen.c c_flags c_library_flags)
# 14 |  (action
# 15 |   (bash
# 16 |     "%{cc} bindings_stubs_gen.c -I `dirname %{lib:ctypes:ctypes_cstubs_internals.h}` -I %{ocaml_where} $(< c_flags) $(< c_library_flags) -o %{targets}")))
# (cd _build/default/stubgen && /bin/bash -e -u -o pipefail -c 'gcc -O2 -fno-strict-aliasing -fwrapv -pthread -fPIC -D_FILE_OFFSET_BITS=64 bindings_stubs_gen.c -I `dirname /home/opam/.opam/4.14/lib/ctypes/ctypes_cstubs_internals.h` -I /home/opam/.opam/4.14/lib/ocaml $(< c_flags) $(< c_library_flags) -o bindings_stubs_gen.exe')
# In file included from /usr/include/guile/3.0/libguile/array-handle.h:27,
#                  from /usr/include/guile/3.0/libguile.h:32,
#                  from bindings_stubs_gen.c:2:
# /usr/include/guile/3.0/libguile/numbers.h:38:10: fatal error: gmp.h: No such file or directory
#    38 | #include <gmp.h>
#       |          ^~~~~~~
# compilation terminated.
# File "lib/dune", line 9, characters 9-20:
# 9 |   (names guile_stubs)
#              ^^^^^^^^^^^
# (cd _build/default/lib && /usr/bin/gcc -I/usr/include/guile/3.0 -I/usr -g -I /home/opam/.opam/4.14/lib/ocaml -I /home/opam/.opam/4.14/lib/base/caml -I /home/opam/.opam/4.14/lib/bigarray-compat -I /home/opam/.opam/4.14/lib/ctypes -I /home/opam/.opam/4.14/lib/integers -I /home/opam/.opam/4.14/lib/ocaml/threads -I /home/opam/.opam/4.14/lib/parsexp -I /home/opam/.opam/4.14/lib/sexplib -I /home/opam/.opam/4.14/lib/sexplib0 -I /home/opam/.opam/4.14/lib/stdlib-shims -I ../bindings -o guile_stubs.o -c guile_stubs.c)
# In file included from /usr/include/guile/3.0/libguile/array-handle.h:27,
#                  from /usr/include/guile/3.0/libguile.h:32,
#                  from guile_stubs.c:20:
# /usr/include/guile/3.0/libguile/numbers.h:38:10: fatal error: gmp.h: No such file or directory
#    38 | #include <gmp.h>
#       |          ^~~~~~~
# compilation terminated.
```
```suggestion
  "conf-guile" {>= "1"}
  "conf-gmp" {>= "1"} # prevents a bug on alpine. See https://github.com/ocaml/opam-repository/pull/21652
```